.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,83 @@
+name: Bug Report
+description: Report a bug encountered while operating Tetragon
+labels: ["kind/community-report", "kind/bug", "needs/triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If you have usage questions, please try the [slack channel](https://cilium.herokuapp.com/) and see the [FAQ](https://github.com/cilium/tetragon#faq) first.
+
+        **Important**: For security related issues: We strongly encourage you to report security vulnerabilities [here](https://github.com/cilium/tetragon/security/advisories/new) - first, before disclosing them in any public forums.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: tetra-version
+    attributes:
+      label: Tetra Version
+      description: What version of the software was running when you discovered this issue? (run `tetra version`)
+    validations:
+      required: true
+  - type: textarea
+    id: kernel-version
+    attributes:
+      label: Kernel Version
+      description: Which kernel version was Cilium running on? (run `uname -a`)
+    validations:
+      required: true
+  - type: textarea
+    id: k8s-version
+    attributes:
+      label: Kubernetes Version
+      description: Which Kubernetes version are you running? (run `kubectl version`)
+    validations:
+      required: true
+  - type: textarea
+    id: sysdump
+    attributes:
+      label: Sysdump
+      description: |
+        - [Install Cilium CLI](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/#install-the-cilium-cli).
+        - Run `cilium sysdump` command
```

Comment:
I think we can use `tetra bugtool` in place of `cilium sysdump`, cc @kkourt knows about this more than me.